src/core/Akka/Actor/Props.cs

```diff
@@ -63,8 +63,9 @@ protected Props()
         /// </summary>
         /// <param name="copy">The object that is being cloned.</param>
         protected Props(Props copy)
-            : this(copy.Deploy, copy._inputType, copy.SupervisorStrategy, copy.Arguments)
+            : this(copy.Deploy, copy._inputType, copy.Arguments)
         {
+            SupervisorStrategy = copy.SupervisorStrategy;
```

Comment:
Should we also copy over the producer at this step?

Reply:
Yes, we should.